src/applications/burials/helpers.jsx

```diff
@@ -163,18 +163,32 @@ export const transportationWarning = (
   </div>
 );
 
-export const burialDateWarning = (
-  <div className="usa-alert usa-alert-warning background-color-only">
-    <span>
-      If filing for a non-service-connected allowance, the Veteran’s burial date
-      must be no more than 2 years from the current date. Find out if you still
-      qualify.{' '}
-      <a href="/burials-memorials/eligibility/" target="_blank">
-        Learn about eligibility
-      </a>
-    </span>
-  </div>
-);
+export const BurialDateWarning = () => {
+  const [shouldRender, setShouldRender] = useState(false);
+
+  useEffect(() => {
+    // NOTE: If we don't wait at least 900ms to render,
+    // the alert content gets overridden by the year content.
+    // Using 1000ms to give a bit of padding
+    const timeout = setTimeout(() => setShouldRender(true), 1000);
```

Comment:
I wonder if 1000ms is enough for a phone with a slow processor? Not sure how to test for this...